compiler/record-minmax/CMakeLists.txt

```diff
@@ -23,6 +23,24 @@ if(NOT ENABLE_TEST)
   return()
 endif(NOT ENABLE_TEST)
 
+# create record-minmax-for-thread-test target
+add_executable(record-minmax-for-thread-test ${DRIVER} ${SOURCES})
```

Comment:
Q)why is `record-minmax-for-thread-test` created with same files as `record-minmax` ?

Reply:
`record-minmax-for-thread-test` is built with `-fsanitize=thread` so that thread sanitizer can check memory bugs. `record-minmax` is built without the option for performance.

Reply:
please add a comment about this reason

Reply:
> please add a comment about this reason

done